music21/environment.py

```diff
@@ -1462,7 +1465,7 @@ def testToSettings(self):
   <preference name="braillePath" />
   <preference name="debug" value="0" />
   <preference name="directoryScratch" />
-  <preference name="graphicsPath" value="/Applications/Preview.app" />
+  <preference name="graphicsPath" value="/System/Applications/Preview.app" />
```

Comment:
I don't think this is the correct path for macOS Preview.app

Reply:
Actually, nevermind. I'm completely wrong. I hadn't noticed that Apple changed this around. Sorry.

Reply:
No problem! In 6.1 there is a new function to derive the path to Preview based on the macOS version, but these tests had not been updated, so they would happen to pass on Travis CI but fail on my Mac.